disperser/dataapi/server.go

```diff
@@ -859,6 +872,74 @@ func (s *server) FetchRegisteredOperators(c *gin.Context) {
 	})
 }
 
+// FetchOperatorEjections godoc
+//
+//	@Summary	Fetch list of operator ejections over last N days.
+//	@Tags		OperatorsInfo
+//	@Produce	json
+//	@Param		days		query		int		false	"Lookback in days [default: 1]"
+//	@Param		operator_id	query		string	false	"Operator ID filter [default: all operators]"
+//	@Param		first		query		int		false	"Return first N ejections [default: 1000]"
```

Comment:
I'm not sure we want to expose these two at API, they can be internal controls

Reply:
The reason subgraph has these is to force pagination on the caller to limit response from getting too big. I think we should pass pagination onto our callers as well. We shouldn't do pagination for them and thus return a huge response payload.
I considered setting internal limits and not exposing these, but it seems like we should allow people to find every ejection that has ever occurred - in which case we should just provide them the tools to do it.

Reply:
This deviates from the existing API norms and creates inconsistent API experience.
Adding more parameters also complicates things, even this implementation of pagination is not quite correct (a second call will is not strictly relative to the first one). I would prefer a simpler semantics with a caveat that it'll provide at most MIN{N, pastNumHoursWindow} items.